content/docs/install/_index.md

```diff
@@ -65,9 +65,9 @@ The [OSM chart](https://github.com/openservicemesh/osm/tree/release-v0.8/charts/
 You can configure the OSM installation by overriding the values file.
 1. Create a copy of the [values file](https://github.com/openservicemesh/osm/blob/release-v0.8/charts/osm/values.yaml) (make sure to use the version for the chart you wish to install).
 1. Change any values you wish to customize. You can omit all other values.
-   - To see which values correspond to the ConfigMap settings, see the [OSM ConfigMap documentation](https://github.com/openservicemesh/osm/blob/main/docs/content/docs/osm_config_map.md)
+   - To see which values correspond to the MeshConfig settings, see the [OSM MeshConfig documentation](https://github.com/openservicemesh/osm-docs/content/docs/osm_mesh_config.md)
```

Comment:
To link to the docs website that you update here, you want this:  \[OSM MeshConfig documentation](/docs/osm_mesh_config)

Reply:
thanks, I have updated the links